libs/clients/user-notification/src/lib/userNotificationClient.module.ts

```diff
@@ -1,8 +1,11 @@
 import { Module } from '@nestjs/common'
-import { UserNotificationApiProvider } from './apiConfiguration'
+import {
+  UserNotificationApiProvider,
+  NotificationsApiProvider,
+} from './apiConfiguration'
 
 @Module({
-  providers: [UserNotificationApiProvider],
-  exports: [UserNotificationApiProvider],
+  providers: [UserNotificationApiProvider, NotificationsApiProvider],
```

Comment:
By using the config optional functionality for the system notification config
https://github.com/island-is/island.is/tree/main/libs/nest/config#optional-config

I'm wondering if that will help us not needing to import it in all the modules that are only using the user specific api, and similar for the auth api lib and then only needing it in the delegation api, as we know that is the only api of the auth apis using this.